ddl/column_change_test.go

```
@@ -0,0 +1,271 @@
+package ddl
+
+import (
+	"fmt"
+
+	"github.com/juju/errors"
+	. "github.com/pingcap/check"
+	"github.com/pingcap/tidb/ast"
+	"github.com/pingcap/tidb/context"
+	"github.com/pingcap/tidb/kv"
+	"github.com/pingcap/tidb/meta"
+	"github.com/pingcap/tidb/meta/autoid"
+	"github.com/pingcap/tidb/model"
+	"github.com/pingcap/tidb/table"
+	"github.com/pingcap/tidb/util/testleak"
+	"github.com/pingcap/tidb/util/testutil"
+	"github.com/pingcap/tidb/util/types"
+)
+
+var _ = Suite(&testColumnChangeSuite{})
+
+type testColumnChangeSuite struct {
+	store  kv.Storage
+	dbInfo *model.DBInfo
+}
+
+func (s *testColumnChangeSuite) SetUpSuite(c *C) {
+	s.store = testCreateStore(c, "test_column_change")
+	s.dbInfo = &model.DBInfo{
+		Name: model.NewCIStr("test_column_change"),
+		ID:   1,
+	}
+	err := kv.RunInNewTxn(s.store, true, func(txn kv.Transaction) error {
+		t := meta.NewMeta(txn)
+		err1 := errors.Trace(t.CreateDatabase(s.dbInfo))
```

Why not return the error directly?